robots.txt.sample

```diff
@@ -59,3 +71,32 @@ Disallow: /*?q=*
 Disallow: /*?p=*&
 Disallow: /*.php$
 Disallow: /*?SID=
+
+############################################
+## Do not allow media indexing for the following bots
+## Disallow all or add custom paths. For example */media/ or */skin/
+
+# User-agent: baiduspider-image
+# Disallow: /
+# Disallow: */media/
+# Disallow: */skin/
+
+# User-agent: baiduspider-video
+# Disallow: /
+# Disallow: */media/
+# Disallow: */skin/
+
+# User-agent: msnbot-media
+# Disallow: /
+# Disallow: */media/
+# Disallow: */skin/
+
+# User-agent: Googlebot-Image
+# Disallow: /
```

Comment:
some people might rely on their product and other images being available via image search

Reply:
This is the main reason those line are commented from the very beginning. But there are situations when you have a website and maybe you don't want your images to be indexed even by Googlebot-Image. in this case you uncomment on those lines.